src/types.ts

```diff
@@ -286,13 +287,16 @@ export type ChannelMemberResponse<UserType = UR> = {
 
 export type CheckPushResponse = APIResponse & {
   device_errors?: {
-    error_message?: string;
-    provider?: string;
+    [deviceID: string]: {
```

Comment:
Have we had the wrong structure all this time ?

Reply:
Yes, it seems type was wrong. It's not a single one but a map of errors with their ids